crates/dscvr-telemetry-util/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+use axum::{extract::MatchedPath, middleware::Next, response::Response};
```

Comment:
Do we also want a method to setup the recorder as described here: fn setup_metrics_recorder() -> PrometheusHandle {?